docs/cep/reference/extensions/index.md

```diff
@@ -0,0 +1,86 @@
+---
+sidebar_position: 3
+title: Extensions
+---
+
+# Extensions
+
+Extensions enhances Macrometa Stream QL by incorporating additional functionalities in a seamless manner.
```

Comment:
What is Stream QL? Is it another name for stream workers?

Reply:
@diana-macrometa - 
- `Stream QL`  intentavely refers to `SQL`. This is the language the customers use writing stream workers and it is very close to `SQL`.
- The extensions allow customers to use different sources, sinks (e.g. GDN Streams, Kafka, ActiveMQ) and the appropriate mapping for them (e.g. JSON, XML, CSV) as well as many functions for different kind of processing (e.g. `Math`, `String`, `Statistics`).

Reply:
Do we actually talk about Stream QL anywhere? This is the first time I remember seeing it. I'll take a stab at some language based on what you said.

Reply:
Yes, that's new for me as well @diana-macrometa but looks good as interpretation.